From 194e09bd80c96bd3826b455e8c38371770889217 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Fri, 16 Sep 2011 18:51:21 -0700
Subject: [PATCH:libXi] Make shadow man pages generated by asciidoc work with Solaris man

Solaris man requires .so entries in man pages include the man section
directory, not just the man page filename, even when it's in the same
directory, so use sed to add it in.

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 man/Makefile.am |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index d4c37b3..28211a1 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -159,6 +159,9 @@ MAN_SUBSTS += -e 's/libmansuffix/$(LIB_MAN_SUFFIX)/g'
 MAN_SUBSTS += -e 's/\[FIXME: source\]/$(XORG_MAN_PAGE)/'
 MAN_SUBSTS += -e 's/\[FIXME: manual\]/XINPUT FUNCTIONS/'
 
+# asciidoc generates shadow page references without the man section directory
+MAN_SUBSTS += -e 's/^\.so X/.so man$(LIB_MAN_SUFFIX)\/X/'
+
 SUFFIXES = .man .$(LIB_MAN_SUFFIX)
 .man.$(LIB_MAN_SUFFIX):
 	$(AM_V_GEN)$(SED) $(MAN_SUBSTS) < $< > $@
-- 
1.7.3.2

